chainntnfs/txnotifier.go

```diff
@@ -887,29 +907,50 @@ func (n *TxNotifier) dispatchConfDetails(
 	return nil
 }
 
+// validateSpendRegistration validates all of the parameters required to
+// successfully register a spend notification.
+func (n *TxNotifier) validateSpendRegistration(outpoint *wire.OutPoint,
+	pkScript []byte, heightHint uint32) (*SpendNtfn, error) {
```

Comment:
Same comment about function name. Maybe `createSpendNtfn` describes the main purpose better, even if we later add some validation in here.

Reply:
Done.